cmd/cstor-pool-mgmt/controller/pool-controller.go

```diff
@@ -0,0 +1,357 @@
+/*
```

Comment:
IMO file name should be `pool_controller.go` as i see in k8s code base and go naming standards.

Reply:
But the name of application/package itself is cstor-pool-mgmt, right?

Reply:
pkg name is fine, it should be `name-verb` , but go files should be in ` _` format

Reply:
ok, will change